libs/designsystem/src/lib/components/checkbox/checkbox.component.scss

```
@@ -70,7 +70,7 @@ $default-checkbox-radio-size: map.get(utils.$checkbox-radio-sizes, 'md');
       --background: #{interaction-state.get-state-color('white', 's')};
     }
 
-    --transition: #{utils.get-transition-duration('quick')};
+    --transition: utils.get-transition-duration('quick');
```

Isn't interpolation required when assigning "Sass members" as values to CSS custom properties?

```suggestion
    --transition: #{utils.get-transition-duration('quick')};
```